platform.md

```diff
@@ -499,11 +500,13 @@ Usage:
 | `<log-level>`            | `CNB_LOG_LEVEL`           | `info`                   | Log Level                                                                                         |
 | `<skip-layers>`          | `CNB_SKIP_LAYERS`         | `false`                  | Do not perform [layer restoration](#layer-restoration)                                            |
 | `<uid>`                  | `CNB_USER_ID`             |                          | UID of the build image `User`                                                                     |
-
+| `<run>`**                | `CNB_RUN_PATH`            | `/cnb/run.toml`          | Path to run file (see [`run.toml`](#runtoml-toml))                                                |
 > ^ Only needed when using image extensions
 
 > \* Only needed when using image extensions to extend the build image
 
+> \** Only needed when using image extensions to extend the run image
```

Comment:
Do we want to add something about how it's used? Looking at this block:

```
- The lifecycle:
  - MUST resolve `run-image.reference` to a digest reference in `<analyzed>` if not present
```

Do we want to make it something like:

```
- The lifecycle:
  - MUST [resolve mirrors](https://github.com/buildpacks/spec/blob/main/platform.md#run-image-resolution) for the `run-image.reference` in `<analyzed>` and resolve it to a digest reference
```